packages/application-shell/src/constants.ts

```diff
@@ -37,6 +37,8 @@ export const STORAGE_KEYS = {
   SELECTED_DATA_LOCALE: 'selectedDataLocale',
   IS_FORCED_MENU_OPEN: 'isForcedMenuOpen',
   LOGIN_STRATEGY: 'loginStrategy',
+  IS_NEW_DESIGN_RELEASE_NOTIFICATION_DIALOG:
+    'isNewDesignReleaseNotificationDialog',
```

Comment:
Nit: let's name it as a status

```suggestion
  IS_NEW_DESIGN_RELEASE_NOTIFICATION_CLOSED:
    'isNewDesignReleaseNotificationClosed',
```

Reply:
Thanks, updated here: https://github.com/commercetools/merchant-center-application-kit/pull/3065/commits/6ba8ec2feb5a111e89264ca8ef8f57cf3b593cb6